app/models/mixins/relationship_mixin.rb

```diff
@@ -570,6 +570,7 @@ def parent=(parent)
     else
       parent.with_relationship_type(relationship_type) do
         parent_rel = parent.init_relationship
+        save!
```

Comment:
yes Keenan it's necessary, see https://github.com/ManageIQ/manageiq/pull/20274/files#r440478954

Reply:
think this is from the change in `save_genealogy_information`

rolling that back will simplify this PR and not introduce that gook.

The "api" for this assumes it is saved later.